lib/initialize.ts

```diff
@@ -22,8 +22,23 @@ export default function createInitializer(
 
   return function init(
     initCb: (sdk: KnownSDK, customSdk: any) => any,
-    { makeCustomApi }: { makeCustomApi?: Function } = {}
+    {
+      makeCustomApi,
+      supressIframeWarning
+    }: { makeCustomApi?: Function; supressIframeWarning?: boolean } = {
+      supressIframeWarning: false
+    }
   ) {
+    if (!supressIframeWarning && currentWindow.self === currentWindow.top) {
+      console.error(`Cannot use ui-extension-sdk outside of Contenful:
+
+It appears that you are trying to use the ui-extension-sdk outside of Contentful
+
+In order for the ui-extension-sdk to function correctly, your app needs to be run in an iframe in the Contentful webapp
```

Comment:
In the FAQ we use "Web App" instead of webapp.

```suggestion
In order for the ui-extension-sdk to function correctly, your app needs to be run in an iframe in the Contentful Web App
```